ui/apps/dashboard/src/components/Metrics/ActionMenu.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import { Listbox } from '@headlessui/react';
+import { NewButton } from '@inngest/components/Button';
+import { Switch } from '@inngest/components/Switch';
+import { useSearchParam } from '@inngest/components/hooks/useSearchParam';
+import { RiSettingsLine } from '@remixicon/react';
+
+export const AUTO_REFRESH_INTERVAL = 5;
+
+export const MetricsActionMenu = () => {
+  const [autoRefresh, setAutoRefresh, removeAutoRefresh] = useSearchParam('autoRefresh');
+
+  return (
+    <Listbox>
+      <Listbox.Button as="div">
+        <NewButton
+          kind="secondary"
+          appearance="outlined"
+          size="medium"
+          icon={<RiSettingsLine />}
+          className="text-sm"
+        />
+      </Listbox.Button>
+      <div className="relative">
+        <Listbox.Options className="bg-canvasBase border-subtle shadow-tooltip absolute right-1 top-1 z-50 h-[52px] w-[247px] gap-y-0.5 rounded border shadow-2xl">
+          <Listbox.Option
+            className="text-subtle mx-2 mt-2 flex cursor-pointer flex-row items-center justify-between text-[13px]"
```

Comment:
I've been seeing a lot of designs these days with 13px for fonts. Perhaps it wouldn't be such a bad idea to create a custom tailwind class 🤔